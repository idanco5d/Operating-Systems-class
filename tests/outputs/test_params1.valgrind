==7018== Memcheck, a memory error detector
==7018== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==7018== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==7018== Command: /mnt/c/Users/illus/os/hw/1/smash
==7018== Parent PID: 7013
==7018== 
==7018== 
==7018== FILE DESCRIPTORS: 4 open (3 std) at exit.
==7018== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_params1.valgrind
==7018==    <inherited from parent>
==7018== 
==7018== 
==7018== HEAP SUMMARY:
==7018==     in use at exit: 0 bytes in 0 blocks
==7018==   total heap usage: 83 allocs, 83 frees, 90,340 bytes allocated
==7018== 
==7018== All heap blocks were freed -- no leaks are possible
==7018== 
==7018== For lists of detected and suppressed errors, rerun with: -s
==7018== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
