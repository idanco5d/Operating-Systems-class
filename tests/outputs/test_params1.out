smash> submitted by Eran
smash> Two> Name1> Name2> smash pid is 9532
Name2> smash pid is 9532
Name2> /tmp/smash_test
Name2> /tmp/smash_test
Name2> smash> 